internal/symbolz/symbolz.go

```diff
@@ -63,10 +63,29 @@ func Symbolize(p *profile.Profile, force bool, sources plugin.MappingSources, sy
 	return nil
 }
 
+// Check whether path ends with one of the suffixes listed in
+// pprof_remote_servers.html from the gperftools distribution
+func hasLegacySuffix(path string) bool {
+	var suffixes = [...]string{
```

Comment:
Can be const:
```
const suffixes := []string{
  ...
}
```

Reply:
It seems arrays and slices cannot be made const:
```
internal/symbolz/symbolz.go:69:8: const initializer [5]string literal is not a constant
```

Reply:
Const arrays (and slices) are discussed here: https://github.com/golang/go/issues/6386